.github/workflows/go.yml

```diff
@@ -29,6 +29,13 @@ jobs:
     - name: Run unit tests
       run: make test-unit
 
+    - name: Codecov
+      uses: codecov/codecov-action@v1.0.13
```

Comment:
why is the action version different here from the one below (integration.yml)?

Reply:
Basically the same, but this version has some enhancements.
I would use v1.0.13 for both of them.